bookingticketui/src/components/Search/searchModel.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useState, useEffect, Fragment } from "react";
+import { useNavigate } from "react-router-dom";
+import { getAirports, getSearch } from "../../services/search-services";
+import * as Yup from "yup";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const searchModel = ({ dataSubmit, setResponse, response }) => {
+  const [boxvalue, setBoxvalue] = useState([]);
+  const navigate = useNavigate();
+  useEffect(() => {
+    (async () => {
+      await getAirports().then((res) => setBoxvalue(res.data.result));
+    })();
+  }, []);
+
+  const validationSchema = Yup.object().shape({
+    depart: Yup.string().required("Depart is required"),
+    arrival: Yup.string()
+      .notOneOf([Yup.ref("depart"), null], "Arrial must not same depart")
```

Comment:
Typo mistake here. It should be "arrival" instead of "arrival".

Reply:
fix